supportedBackends/google/batch/src/test/scala/cromwell/backend/google/batch/api/BatchRequestExecutorSpec.scala

```diff
@@ -0,0 +1,75 @@
+package cromwell.backend.google.batch.api.request
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import com.google.cloud.batch.v1.{
+  AllocationPolicy,
+  BatchServiceClient,
+  BatchServiceSettings,
+  GetJobRequest,
+  Job,
+  JobStatus
+}
+import com.google.cloud.batch.v1.AllocationPolicy.{
+  InstancePolicy,
+  InstancePolicyOrTemplate,
+  LocationPolicy,
+  ProvisioningModel
+}
+import common.mock.MockSugar
+import cromwell.backend.google.batch.api.BatchApiResponse
+import cromwell.backend.google.batch.models.RunStatus
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.doReturn
+import org.scalatest.PrivateMethodTester
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+
+class BatchRequestExecutorSpec
+    extends TestKit(ActorSystem("BatchRequestExecutorSpec"))
+    with AnyFlatSpecLike
+    with Matchers
+    with MockSugar
+    with PrivateMethodTester {
+
+  behavior of "BatchRequestExecutor"
+
+  it should "create instantiatedVmInfo correctly" in {
+
+    val instancePolicy = InstancePolicy
+      .newBuilder()
+      .setMachineType("n1-standard-1")
+      .setProvisioningModel(ProvisioningModel.PREEMPTIBLE)
+      .build()
+
+    val allocationPolicy = AllocationPolicy
+      .newBuilder()
+      .setLocation(LocationPolicy.newBuilder().addAllowedLocations("regions/us-central1"))
```

Comment:
I think it would be useful to have some tests checking how we handle unexpected location values - I'm nervous any time we're depending on Google not to change the way they generate a string.

Reply:
yeahhh makes sense, I can add another